### ISSUE TITLE
Need DPub roles for page-level running headers and footers

### ISSUE THREAD

**Issue by aleventhal** [w3c/dpub-aria#10]
Moved from https://github.com/w3c/dpub/issues/24

Without these semantics, screen readers cannot provide their users with settings to turn on/off the visibility of running headers and footers, which may have important information, or the user may wish to examine for authoring purposes.

**Comment by ggordon-vfo** [comment]
Agree wholeheartedly with this proposal. Without this level of granularity,  ATs won't be able to offer a user option to speak header and footer details automatically.

**Comment by joanmarie** [comment]
This seems like a valid thing which should be added to the DPub-ARIA spec.

Note that the ARIA Working Group doesn't own this spec; we collaborate with Publishing WG on it however. See https://www.w3.org/2017/04/publ-wg-charter/#deliverables.

**Comment by aleventhal** [comment]
Right .. what are the next steps tho? I don't think a new version of DPUB ARIA is currently planned?

**Comment by joanmarie** [comment]
I see it on https://www.w3.org/2017/04/publ-wg-charter/#deliverables. I don't know what the status is. Did you talk with the Publishing WG about this?

**Comment by aleventhal** [comment]
No, I haven't -- thanks I'll try that. Makes sense.

**Comment by dauwhe** [comment]
What sort of markup is being used to describe running headers or footers? There are CSS specs that describe such information as generated content that might not appear in an HTML source.
